packages/frint/src/App.ts

```diff
@@ -0,0 +1,410 @@
+import { Container, createContainer, Provider, resolveContainer } from 'frint-di';
+import { find, findIndex, get, omit } from 'lodash';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { concatMap as concatMap$, find as find$, first as first$, map as map$ } from 'rxjs/operators';
```

Comment:
to help reduce bundle size, please consider importing the modules individually:

```js
import { concatMap } from 'rxjs/operators/concatMap';
```

---

It was indeed me that started the pattern of importing operators with a name ending `$`, but I really see no value in doing so now. It was mostly done to avoid conflicts with lodash functions, which I don't remember facing anywhere.